enos/modules/k8s_vault_verify_ui/main.tf

```diff
@@ -0,0 +1,42 @@
+
+terraform {
+  required_providers {
+    enos = {
+      version = ">= 0.1.17"
+      source  = "app.terraform.io/hashicorp-qti/enos"
+    }
+  }
+}
+
+locals {
+  instances = toset([for idx in range(var.vault_instance_count) : tostring(idx)])
+}
+
+resource "enos_remote_exec" "curl_ui" {
+  for_each = local.instances
+
+  inline = [
+    "curl -s -o /dev/null -w '%%{redirect_url}' http://localhost:8200/",
```

Comment:
I'm quite surprised the the docker image that we ship includes `curl`

Reply:
yeah thank god.